build.py

```diff
@@ -1,18 +1,71 @@
-import math
-import subprocess
 from pathlib import Path
 
+import argparse
+
 import fontmake.instantiator
 import fontTools.designspaceLib
 import ufo2ft
 import ufoLib2
 import vttLib
-import copy
 
 INPUT_DIR = Path("sources")
 OUTPUT_DIR = Path("build")
+VTT_DATA_FILE = (INPUT_DIR / "vtt_data" / "CascadiaCode.ttx")
+
+def step_set_font_name(n):
+    def _set(instance):
+        instance.info.familyName = n
+        # We have to change the style map family name because that's what
+        # Windows uses to map Bold/Regular/Medium/etc. fonts
+        instance.info.styleMapFamilyName = n
+    return _set
+
+def step_merge_glyphs_from_ufo(path):
+    def _merge(instance):
+        ufo = ufoLib2.Font.open(path)
+        print (f"[{instance.info.familyName}] Merging {path}")
+        for glyph in ufo.glyphOrder:
+            if glyph not in instance.glyphOrder:
+                instance.addGlyph(ufo.get(glyph))
```

Comment:
I wonder if it's necessary to `instance.lib["public.glyphOrder"].extend(ufo.lib["public.glyphOrder"])` here to make glyph order deterministic :thinking: @anthrotype didn't we talk about this recently?

Reply:
Interesting. We should file a followup on this.